scripts/macosx/build_qt5.sh

```diff
@@ -29,8 +29,6 @@ export DISABLE_FFAST_MATH=yes
 source $PROGDIR/environment.sh
 export QTDIR=$MIXXX_PREFIX/Qt-${VERSION_NUMBER}
 
-echo "Building $VERSION for $MIXXX_ENVIRONMENT_NAME for architectures: ${QT_ARCHS[@]}"
```

Comment:
Why remove this line? It's in every build_ file so I'd prefer to keep it for consistency.

Reply:
It was redundant. See line 18 above.